src/components/marker-chart/Canvas.js

```diff
@@ -220,12 +221,15 @@ class MarkerChartCanvas extends React.PureComponent<Props, State> {
             w = 10;
           }
 
+          x = Math.round(x);
```

Comment:
This is lowering the visual fidelity of the graph. I would suggest rounding the `DevicePixel` rather than the `CssPixel`.

`x = Math.round(x * devicePixelRatio) / devicePixelRatio`